getting-started/quickstart.md

```diff
@@ -130,17 +130,17 @@ However, if you set up an organization, create the repo under the aegis of the o
 
 ### FTP Deployment
 
-It is possible to deploy via FTP with PHP applications.  
+You can deploy via FTP with PHP applications.  
 
 To deploy via FTP, you need an FTP software installed on your machine. [Filezilla](https://filezilla-project.org/) is one of them.
 
-When you have chosen to deploy your application via FTP, a [FS Bucket]({{< ref "/deploy/addon/fs-bucket.md" >}}) has been created with an ID
+Deploy your application via FTP, create a [FS Bucket]({{< ref "/deploy/addon/fs-bucket.md" >}}) with an ID
```

Comment:
Actually, the FSBucket is automatically created when you create an FTP application, you don't need to create one.